src/App/ImportUshahidiV2/Mappers/IncidentPostMapper.php

```diff
@@ -72,4 +87,34 @@ public function getCategories($importId, $categories)
             return $this->mappingRepo->getDestId($importId, 'category', $item);
         })->all();
     }
+
+    public function getMedia($media, $type, $user_id)
+    {
+        return collect($media)
+            ->where('media_type', $type)
+            ->map(function ($media) use ($type, $user_id) {
+                // Not sure what to do with non URL values yet, so just saving them as-is
```

Comment:
This ^^. I'm not copying the media. Most of Crowdmap references an absolute URL on the CDN, so we can probably use that. Or do a manual copy for non-crowdmap deployments?